source/docs/yearly-overview/known-issues.rst

```diff
@@ -8,6 +8,17 @@ This article details known issues (and workarounds) for FRC\ |reg| Control Syste
 Open Issues
 -----------
 
+No such host is known in the WPILib Installer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following error message will show up in installs when downloading VS Code before version ``2021.2.2`` **and** ``3/24/2021``.
```

Comment:
```suggestion
The following error message will show up in installs when downloading VS Code with WPILib Installer version ``2021.2.1`` or earlier **or** version ``2021.2.2`` that was downloaded before ``3/24/2021``.
```

Reply:
I'm not sure this conveys it any more clear?